test/rsa_genrsa_check.sh

```diff
@@ -48,8 +48,8 @@ openssl rsa -RSAPublicKey_in -in testrsa.pem -modulus -noout
 openssl rsa -RSAPublicKey_in -in testrsa.pem -text -noout
 
 
-# read DER and export public key as DER
-openssl rsa -provider tpm2 -provider base -in pubkey.pem -inform der -RSAPublicKey_out -outform der -out testrsa.der
+# read PEM and export public key as DER
+openssl rsa -provider tpm2 -provider base -in pubkey.pem -inform pem -RSAPublicKey_out -outform der -out testrsa.der
```

Comment:
The idea here was to try all combinations PEM-PEM, PEM-DER, DER-PEM and DER-DER. The comment was correct, even the (useless) `-inform` argument was right-- it's the `-in pubkey.pem` that is wrong here. That should be `-in pubkey.der`.

Reply:
Okie!  Fixed, and commit message changed accordingly